app/core/ai_monitoring.py

```
@@ -0,0 +1,186 @@
+        # Anomaly detection thresholds
+        self._conn.execute("""
+            CREATE TABLE IF NOT EXISTS anomaly_thresholds (
+                metric_name TEXT PRIMARY KEY,
+                lower_bound REAL,
+                upper_bound REAL,
+                last_updated TIMESTAMP
+            )
+        """)
+
+    def record_metric(self, metric_name: str, value: float, context: Optional[Dict] = None):
+        timestamp = datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)